cnxdb/archive-sql/get-featured-links.sql

```diff
@@ -17,11 +17,11 @@ FROM (SELECT
     a.html AS abstract,
     '/resources/' || f.sha1 AS "resourcePath",
     t.tag AS "type"
-FROM modules m
-  LEFT JOIN moduletags mt ON m.module_ident = mt.module_ident
-  LEFT JOIN tags t ON mt.tagid = t.tagid
-  LEFT JOIN abstracts a ON m.abstractid = a.abstractid
-  LEFT JOIN module_files mf ON mf.module_ident = m.module_ident
-  LEFT JOIN files f ON f.fileid = mf.fileid
-WHERE t.scheme = 'featured' AND mf.filename = 'featured-cover.png'
+FROM featured_books as f
+  LEFT JOIN modules as m
+    ON (f.uuid=m.uuid
+    and (f.major_version=m.major_version OR
+         f.major_version is NULL)
+    and (f.minor_version=m.minor_version OR
+         f.minor_version is NULL))
```

Comment:
Hmm, we need to decide on the user-story - is a featured book most-recent-version-of-a-given-uuid? Then we don't need the version fields at all, and we need to join against latest_modules. If we allow for pointing to a specific older version, then there will be both versions - only individual pages have major-only.